src/main/java/az/edu/turing/entity/Human.java

```diff
@@ -46,16 +48,16 @@ private long parseBirthDate(String birthDate) {
     }
 
     public String describeAge() {
```

Comment:
Pls test this method, you try to pass ofEpochDay method millis, which I think shouldn't work properly